org.gnome.Hexgl.json

```diff
@@ -0,0 +1,78 @@
+{
+    "app-id" : "org.gnome.Hexgl",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gnome-hexgl",
+    "tags" : [
+        "nightly"
+    ],
+    "desktop-file-name-prefix" : "(Nightly) ",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "cflags" : "-O2 -g",
+        "cxxflags" : "-O2 -g",
+        "env" : {
+            "V" : "1"
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "graphene",
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts" : ["-Dman=false"],
```

Comment:
Graphene doesn't have any man page. :wink: 

You probably want: `["-Dgtk_doc=false", "-Dgobject_types=true"]` instead.

Reply:
Yes, done!